lib/hangouts_chat.rb

```diff
@@ -19,19 +19,19 @@ def initialize(webhook_url)
     # Sends Simple Text Message
     # @param text [String] text to send to room
     # @return [Net::HTTPResponse] response object
-    def simple(text)
+    def simple(text, thread: nil)
```

Comment:
Great, thank you!

Can you please also add `@param` description into methods documentation?

Reply:
I had missed this.
Committed the fix (force-push it).